demos/index.html

```diff
@@ -8,37 +8,38 @@
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/overviewPanel/css/OverviewPanel.css" />
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css" />
 
-        <script type="text/javascript" src="../src/lib/infusion/infusion-custom.js"></script>
-        <script type="text/javascript" src="../src/lib/d3/d3.js"></script>
-
-        <script type="text/javascript" src="../src/lib/flocking/flocking-base.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/oscillators.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/math.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/envelopes.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/midi.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/scheduling.js"></script>
-
-        <script type="text/javascript" src="../src/js/valueBinding.js"></script>
-        <script type="text/javascript" src="../src/js/percentage.js"></script>
-        <script type="text/javascript" src="../src/js/transformations.js"></script>
-        <script type="text/javascript" src="../src/js/templateInjection.js"></script>
-        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
-        <script type="text/javascript" src="../src/js/totalRelaying.js"></script>
-        <script type="text/javascript" src="../src/js/legend.js"></script>
-        <script type="text/javascript" src="../src/js/pie.js"></script>
-        <script type="text/javascript" src="../src/js/pieChart.js"></script>
-        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
-        <script type="text/javascript" src="../src/js/utils.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntry.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
-        <script type="text/javascript" src="../src/js/electricPianoBand.js"></script>
-        <script type="text/javascript" src="../src/js/unitDivisorSonificationStrategy.js"></script>
-        <script type="text/javascript"
-        <script type="text/javascript" src="../src/js/sonifier.js"></script>
-        <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
-
-        <script type="text/javascript" src="src/js/chartAuthoring-demo-overview.js"></script>
-        <script type="text/javascript" src="src/js/chartAuthoring-demo.js"></script>
+        <script  src="../src/lib/infusion/infusion-custom.js"></script>
```

Comment:
I've updated to the latest Infusion and fixed a few issues that arose from model relay and differences in locate/container (I believe the latter were due to issues in the code that were missed because of `{that}.locate(selector)` returning the parent container under previous versions if the DOM element of the selector didn't exist).

I am seeing an issue that appears to originate in Flocking under the latest Infusion, which is causing the Sonifier tests to fail (and the Chart Authoring ones as well, since they use the Sonifier). Easiest to see if you run the Sonifier tests, consistent error is `Failed to resolve reference {scriptProcessor} - could not match context with name scriptProcessor from component { typeName: "flock.webAudio.outputManager" gradeNames`:

```
10:13:21.851:   ASSERTION FAILED:  Failed to resolve reference {scriptProcessor} - could not match context with name scriptProcessor from component { typeName: "flock.webAudio.outputManager" gradeNames: ["flock.outputManager","flock.webAudio.outputManager"] id: bduu0q4i-1421} at path speedySonifier-bduu0q4i-1205.enviro.audioSystem.outputManager component:  
```

@colinbdclark can you comment on Flocking's compatibility with latest Infusion? At first glance the error doesn't seem to originate in the chartAuthoring code.